app/services/sufia/query_service.rb

```diff
@@ -0,0 +1,47 @@
+module Sufia
+  class QueryService
+    # query to find generic files created during the time range
+    # @param [DateTime] start_datetime starting date time for range query
+    # @param [DateTime] end_datetime ending date time for range query
+    def find_by_date_created(start_datetime, end_datetime = nil)
+      return [] if start_datetime.blank? # no date just return nothing
+      klass.where(build_date_query(start_datetime, end_datetime))
+    end
+
+    def where_public
+      where_access_is 'public'
+    end
+
+    def where_registered
+      where_access_is 'registered'
+    end
+
+    def build_date_query(start_datetime, end_datetime)
+      start_date_str =  start_datetime.utc.strftime(date_format)
+      end_date_str = if end_datetime.blank?
+                       "*"
+                     else
+                       end_datetime.utc.strftime(date_format)
+                     end
+      "system_create_dtsi:[#{start_date_str} TO #{end_date_str}]"
+    end
+
+    delegate :count, to: :klass
```

Comment:
My intention is to just keep the existing behavior for now, but to position someone to refactor this class down the road such that it will query every class in `CurationConcerns.config.curation_concerns`